tasks/package.js

```diff
@@ -117,25 +192,31 @@ module.exports = function(grunt) {
       tasks.push('composer:drupal-scaffold');
     }
 
-    if (this.args[0] && this.args[0] === 'compress') {
+    if (archive || (this.args[0] && this.args[0] === 'compress')) {
       grunt.loadNpmTasks('grunt-contrib-compress');
       grunt.config('compress.package', {
         options: {
           archive: destPath + '.tgz',
-          mode: 'tgz',
-          gruntLogHeader: false
+          mode: 'tgz'
         },
         files: [
           {
             expand: true,
             dot: true,
-            cwd: grunt.config.get('config.buildPaths.packages') + '/' + packageName,
+            cwd: destPath,
             src: ['**']
           }
         ]
       });
 
       tasks.push('compress:package');
+
+      if (destPath !== finalPath) {
+        grunt.config('shell.mvArchive', {
+          command: 'mv ' + destPath + '.tgz ' + finalPath + '.tgz'
```

Comment:
So that's what I thought too.  But it seems to work across docker volumes.  Maybe we should change it from a "mv" to a normal grunt copy.  Then delete the tarball from the destPath.